config/linux/ubuntu_install_nsrlsvr.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+#
+# Script to install nsrlsvr on Ubuntu from the GIFT PPA.
+
+AUXILIARY_DATA_PATH="/media/auxiliary";
+
+# Exit on error.
+set -e
+
+# Install and configure nsrlsvr
+sudo apt-get install -y curl nsrlsvr-server unzip
+
+sudo mkdir -p /var/share/nsrlsvr
+
+if [ -f "${AUXILIARY_DATA_PATH}/nsrlsvr/hashes.txt" ];
+then
+	cp -f "${AUXILIARY_DATA_PATH}/nsrlsvr/hashes.txt" /var/share/nsrlsvr;
+fi
+
+if [ ! -f /var/share/nsrlsvr/hashes.txt ];
```

Comment:
is /var/share/nsrlsvr/hashes.txt  normally created by /usr/bin/nsrlupdate NSRLFile.txt ?

Reply:
Correct, I'll add a comment